client/Android/FreeRDPCore/jni/android_freerdp.c

```diff
@@ -705,6 +716,8 @@ JNIEXPORT jboolean JNICALL jni_freerdp_disconnect(JNIEnv *env, jclass cls, jint
 	freerdp* inst = (freerdp*)instance;
 	androidContext* ctx = (androidContext*)inst->context;
 	ANDROID_EVENT* event = (ANDROID_EVENT*)android_event_disconnect_new();
```

Comment:
Since android_event_disconnect_new was modified to return NULL if alloc fails we should check the result here and not rely on the assert below.
And also I'm not sure if the WLog_ERR text below is considered final ;)